pkg/handler/config.go

```diff
@@ -78,6 +78,15 @@ type QuickFilter struct {
 	Default bool `yaml:"default,omitempty" json:"default,omitempty"`
 }
 
+type FieldConfig struct {
+	Name        string `yaml:"name" json:"name"`
+	Type        string `yaml:"type" json:"type"`
+	Description string `yaml:"description" json:"description"`
+	// This flag is for documentation only. Use loki.labels instead
```

Comment:
perhaps it can be omitted then, in this declaration? To avoid any confusion

Reply:
sure :+1: 
https://github.com/netobserv/network-observability-console-plugin/pull/476/commits/2fbb838e1fa395a506aa08ceb12b39f0e8433f87